crates/bevy_ecs/src/world/entity_ref.rs

```diff
@@ -355,28 +386,27 @@ impl<'w> EntityMut<'w> {
                 .tables
                 .get_2_mut(old_table_id, new_archetype.table_id());
 
-            // SAFE: table_row exists. All "missing" components have been extracted into the bundle
-            // above and the caller takes ownership
-            let move_result =
-                unsafe { old_table.move_to_and_forget_missing_unchecked(old_table_row, new_table) };
+            // SAFE: table_row exists
```

Comment:
```suggestion
            // SAFE: old_table_row exists
```

Reply:
I'd prefer to keep the unsafe block around this so then future devs can know to be more cautious here.